<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>other</class>
 <widget class="QWidget" name="other">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="selecttButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="chessButton">
       <property name="text">
        <string>Chessboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="crossButton">
       <property name="text">
        <string>City Block</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="euclideanButton">
       <property name="text">
        <string>Euclidean</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="distanceButton">
       <property name="text">
        <string>Distance Transform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skeletButton">
       <property name="text">
        <string>Skeleton Extraction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restButton">
       <property name="text">
        <string>Skeleton Resoration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edgeButton">
       <property name="text">
        <string>Edge Detection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gradientButton">
       <property name="text">
        <string>Morphological Gradient</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="standardButton">
       <property name="text">
        <string>Standard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="externalButton">
       <property name="text">
        <string>External</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="internalButton">
       <property name="text">
        <string>Internal</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QScrollArea" name="grayArea">
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>437</width>
          <height>530</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Result</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QScrollArea" name="oriArea">
       <property name="widgetResizable">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>438</width>
          <height>530</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Adobe Arabic</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Origin</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
